Workaround: disable el locale (causes po2xml to crash in libpython)
--- gimp-help-2.8.1/configure.ac.orig	2013-11-27 00:28:26.000000000 +0100
+++ gimp-help-2.8.1/configure.ac	2013-12-11 16:29:54.640977970 +0100
@@ -57,12 +57,12 @@
 #  QUICKREFERENCE_ALL_LINGUAS *and* (intersecton) ALL_LINGUAS
 
 if test "x$ALL_LINGUAS" = "x"; then
-  ALL_LINGUAS="ca da de el en en_GB es fr it ja ko nl nn pt_BR ru sl sv zh_CN"
+  ALL_LINGUAS="ca da de en en_GB es fr it ja ko nl nn pt_BR ru sl sv zh_CN"
 else
   ALL_LINGUAS=`echo "$ALL_LINGUAS" | $SED -e 's/[,] */ /g'`
 fi
 if test "x$QUICKREFERENCE_ALL_LINGUAS" = "x"; then
-  QUICKREFERENCE_ALL_LINGUAS="ca de el en fi fr it ja ko ru sv zh_CN"
+  QUICKREFERENCE_ALL_LINGUAS="ca de en fi fr it ja ko ru sv zh_CN"
 fi
 
 AC_SUBST(ALL_LINGUAS)
